make it easier to create empty filesystems (#73)
pengutronix/genimage: README.rst

:mountpoint:	mountpoint if image refers to a filesystem image. The
		default is "/". The content of "${rootpath}${mountpoint}"
		will be used used fill the filesystem.
:empty:		If this is set to true, then the specified rootpath and

> 		mountpoint for the image is ignored for this image and a

Reviewer: ```suggestion
		mountpoint is ignored for this image and an
```

Author: Yes, that sounds better. Fixed.
